meson.build

```diff
@@ -2334,7 +2334,17 @@ subdir('tmpfiles.d')
 subdir('rules')
 subdir('hwdb')
 subdir('network')
-subdir('man')
+
+want_man = get_option('man')
+want_html = get_option('html')
+xsltproc = find_program('xsltproc',
+                        required : want_man == 'true' or want_html == 'true')
+want_man = want_man != 'false' and xsltproc.found()
+want_html = want_html != 'false' and xsltproc.found()
+if want_man or want_html
+        subdir('man')
+endif
```

Comment:
This came up before, and I think it's a bad idea — all directories should be included unconditionally.

What about this instead:
```diff
- foreach tuple : manpages
+ foreach tuple : want_man ? manpages : []
```
?

Reply:
Thanks. Updated.